includes/Experiments.php

```diff
@@ -352,17 +352,6 @@ public function get_experiments() {
 				'description' => __( 'Enable SVG upload', 'web-stories' ),
 				'group'       => 'general',
 			],
-			/**
-			 * Author: @spacedmonkey
-			 * Issue: #7480
-			 * Creation date: 2021-05-28
-			 */
-			[
-				'name'        => 'enableMediaPickerVideoOptimization',
-				'label'       => __( 'Video optimization in media picker', 'web-stories' ),
-				'description' => __( 'Optimize already uploaded videos in media picker.', 'web-stories' ),
-				'group'       => 'editor',
```

Comment:
Confirming -- Is this feature flag meant to be fully removed, or just enabled by default for now (`'default'     => true,`)?

Reply:
I don't see the point of enabling it be default. This makes more sense here, as this is landing early.

Reply:
Ok, that's just what the issue description seems to say. I think @swissspidy mentioned at some point that enabling by default (which forces it to be always enabled and doesn't allow turning off) as the first step is a good practice since this allows reverting the feature quickly if anything goes wrong. Thoughts?

Reply:
What Miina says.

Should be OK in this case, but that two-step approach is a best practice to keep in mind for next time.